src/middleware/api.ts

```diff
@@ -269,3 +301,47 @@ export const refreshAccessTokens = async (): Promise<SessionMeta> => {
 export const hentSesjonMetadata = async (): Promise<SessionMeta> => {
     return fetchToJson('/oauth2/session', {}, false).then(data => Promise.resolve(data as SessionMeta));
 };
+
+export const settBrukerIKontekst = async (fnr: string): Promise<void> => {
+    const respons = await fetch('/modiacontextholder/api/context', {
+        ...MED_CREDENTIALS,
+        method: 'post',
+        body: JSON.stringify({verdi: fnr, eventType: 'NY_AKTIV_BRUKER'})
+    });
+
+    return sjekkStatuskode(respons);
+};
+
+export const hentBrukerIKontekst = async () => {
+    try {
+        const data = await fetchToJson('/modiacontextholder/api/context');
```

Comment:
Based on https://github.com/navikt/modiacontextholder/blob/0cdbbb2cc7507c2e8ea7f55392aa6fc7ca0c7972/src/main/java/no/nav/sbl/rest/ContextRessurs.java#L42 it is necessary to add `/aktivbruker` in endpoint url

Reply:
There are multiple endpoints. `GET /context/aktivbruker` returns only the aktiv bruker, but `GET /context` returns an object with both the aktiv bruker and the aktiv enhet.